web/src/app/unite/services/menus.service.ts

```diff
@@ -0,0 +1,15 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { BaseService } from "./base.service";
+
+@Injectable()
+export class MenusService extends BaseService{
+
+  constructor(_httpClient: HttpClient) {
+    super(_httpClient);
+  }
+
+  getMenus() {
+    return this.get('http://172.132.45.94:3000/api/Menus');
```

Comment:
@ranjit-redekar - get the base path for API from environment variables

Reply:
Ok I will update this in new MR, I have pushed this MR for only for backup.

Reply:
@ranjit-redekar - ok